AMW_angular/io/package.json

```diff
@@ -24,10 +27,10 @@
     "@angular/router": "~10.0.8",
     "@ng-bootstrap/ng-bootstrap": "^7.0.0",
     "@ng-select/ng-select": "~5.0.1",
-    "bootstrap-3-typeahead": "4.0.2",
-    "bootstrap-sass": "3.4.1",
-    "eonasdan-bootstrap-datetimepicker": "4.17.47",
+    "bootstrap": "^4.5.0",
+    "bootstrap-icons": "^1.0.0",
     "jquery": "^3.5.1",
```

Comment:
Should we be able to remove jquery now?

Reply:
no - there are still some modal dialogs that directly use jquery - these could be changed (and I'd be happy to do it) but I think that should go into another issue / pr